### PR TITLE
feat: Add GitHub covid finder to OSS-PORT projects

### PR DESCRIPTION
**Pull request by luisFilipePT**
# CHANGES

Add project to list

### CODE REVIEW

**Review by ramonh — APPROVED** [review]
Thank you so much for adding Github Covid Finder, @luisFilipePT

This looks wonderful! The Map is also so thoughtfully laid out. 

Welcome aboard!

**Comment by luisFilipePT** [comment]
> Thank you so much for adding Github Covid Finder, @luisFilipePT
> 
> This looks wonderful! The Map is also so thoughtfully laid out.
> 
> Welcome aboard!

Thanks, I added a plain map and next I will update it and make it look prettier :)

File: projects/luisFilipePT/github-covid-finder.mdx [hunk 1]
```diff
@@ -16,8 +16,8 @@ currentlySeeking:
   - Developers
   - Frontend Developers
   - Maintainers
-featuredMap: https://app.codesee.io/maps/public/412aa7c0-1d26-11ec-9eb0-ad351a9f5b0c
-  url: https://app.codesee.io/maps/public/848e3630-1650-11ec-8bc1-7d4a4822cc27
+featuredMap:
+  url: https://app.codesee.io/maps/public/412aa7c0-1d26-11ec-9eb0-ad351a9f5b0c
```
Comment (ramonh) [inline projects/luisFilipePT/github-covid-finder.mdx:20]: Out of curiosity, is there a reason you replaced the existing Map with a plain one?

**Comment by ramonh** [comment]
Ah! Disregard my last comment, please. Sounds great! Looking forward. Do give me a shout if I can help with that!

File: projects/luisFilipePT/github-covid-finder.mdx (hunk 1 above)
Comment (luisFilipePT) [reply projects/luisFilipePT/github-covid-finder.mdx:20]: Yes, the existing map was the one from the example 😊 . It was not mine

But I am assuming that now I can have it published and improve the map assuming that the link does not change

Comment (ramonh) [reply projects/luisFilipePT/github-covid-finder.mdx:20]: Ack! My bad, I spoke too soon. 

Fantastic! Indeed the link won't change.

**Pull request merged**